### ISSUE TITLE
add to plugin page

### ISSUE THREAD

**Issue by thenewguy** [heff/videojs-dashjs#1]
If you think the code is ready, it could be added to the videojs plugin wiki https://github.com/videojs/video.js/wiki/Plugins

**Comment by heff** [comment]
Yeah I've been meaning to do that but first I was going to move it into the Dash IF github org, and before doing that I was hoping to get an answer to my question on the dash group about where to put contrib plugins. It doesn't look like I'm gonna get one.
https://groups.google.com/forum/#!topic/dashjs/cDoGtbzm6ro
